_editorial_policies/01_bylaws.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: single
+sidebar:
+  nav: policies_bylaws.md
+title: Bylaws of the Living Journal of Computational Molecular Science
+excerpt: Bylaws of LiveCoMS.
+permalink: /policies/livecoms_bylaws/
+---
+<!-- required to get the labeling right-->
```

Comment:
I first wanted to suggest to put the css commands in the general css file, but I think we can leave this here - the choice of list levels seems to be specific for this file.

Reply:
Oh, I can definitely try putting it there instead.  Probably would be cleaner. We can make consistent list levels.